pensieve/pensieve.py

```diff
@@ -146,10 +146,22 @@ def paragraphs(self):
         if not self._paragraphs:
             print('Generating paragraphs for doc '+str(self.id))
             self._paragraphs = []
-            for i, p in enumerate(self.text.split('\n')):
-                if len(p.split(' ')) < 30:
+            list = self.text.split('\n')
+            i = 0
+            while i < len(list):
```

Comment:
This will be much more readable as a for loop with `enumerate`, rather than a while loop.

You can replace:
```
i = 0
line_list = ['list', 'of', 'stuff']
while i < len(line_list):
    new_list.append(do_stuff(line_list[i], i))
    i += 1
```
with
```
line_list = ['list', 'of', 'stuff']
for i, line in enumerate(line_list):
    new_list.append(do_stuff(line, i))
```